### PR TITLE
AB#892 Create global modal tied with redux

### PR DESCRIPTION
**Pull request by pmakowski23**
By mistake done
https://dev.azure.com/BambitIT/Bambit/_workitems/edit/892/
and
https://dev.azure.com/BambitIT/Bambit/_workitems/edit/889/

### CODE REVIEW

File: src/common/Sandbox/Sandbox.tsx [hunk 1]
```diff
@@ -3,10 +3,21 @@ import Button from '@mui/material/Button';
 import Grid from '@mui/material/Grid';
 
 import { SearchBox } from 'common/components';
+import { openModal } from 'redux/modal';
+import { GlobalModal } from 'common/components/GlobalModal';
+import { useAppDispatch } from 'redux/hooks';
```
Comment (szymonbla) [inline src/common/Sandbox/Sandbox.tsx:8]: I propose to set the import according to the following scheme:
- built-in
- external
- internal

**Review by szymonbla — APPROVED** [review]
One minor suggestion

**Review by mkidawa — APPROVED** [review]
LGTM

**Review by JKrymarys — APPROVED** [review]
Looks good 👍🏻

**Pull request merged**